conda/plugins/hookspec.py

```diff
@@ -332,3 +334,72 @@ def conda_settings():
                    aliases=("example_option_alias",),
                )
         """
+
+    @_hookspec
+    def conda_reporter_backends(self) -> Iterable[CondaReporterBackend]:
+        """
+        Register new reporter backend
+
+        The example below defines a reporter backend that uses the ``pprint`` module in Python.
+
+        **Example:**
+
+        .. code-block:: python
+
+           from pprint import pformat
+
+           from conda import plugins
+           from conda.plugins.types import CondaReporterHandler, ReporterHandlerBase
+
+
+           class PprintReporter(ReporterRendererBase):
+
+               def detail_view(self, data):
+                   return pformat(data)
+
+
+           @plugins.hookimpl
+           def conda_reporter_backends():
+               yield CondaReporterBackend(
+                   name="pprint",
+                   description="Reporter handler based off of the pprint module",
+                   renderer=PprintReporter(),
+               )
+
+        """
+
+    @_hookspec
+    def conda_reporter_streams(self) -> Iterable[CondaReporterStream]:
```

Comment:
Output is the better term for this, as it's generic enough to describe what it does, while the "stream" is the canonical term for a file-like object to write to